.github/workflows/ci.yml

```diff
@@ -365,7 +365,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.9
```

Comment:
@mkundu1 - have a look at other repos. We typically define a MAIN_PYTHON_VERSION. See https://github.com/ansys/pyprimemesh/blob/main/.github/workflows/ci_cd.yml#L19

That way it is easier to maintain and to see what is the main Python version you are doing operations against.

Reply:
Done, Thank you.